docs/.eleventy.js

```diff
@@ -76,6 +77,9 @@ module.exports = function (eleventyConfig) {
   //   }
   // });
 
+  eleventyConfig.on('beforeWatch', () => {
+    child_process.execSync('cat docs/* > index.md').toString('UTF-8')
```

Comment:
any reason to do `toString()` at the end? might be okay to remove (if not, add a comment why)

Reply:
execSync by default is encoded as a buffer which doesn't work but turns out there is a encoding option so I used that instead https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options

Reply:
since we never look at the return value of the function, the code should work the same whether it returns a string or a buffer. 

Not a blocking issue in either case! Just adding for reference